### PR TITLE
CM360 OCI Connector

### PR DESCRIPTION
**Pull request by dandemel**
Merge CM 360 OCI Connector implementation into the main branch

### CODE REVIEW

File: dags/destinations/cm360oci.py [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+        ("client_id", str, Field(description="An OAuth2.0 Web Client ID.")),
+        ("client_secret", str, Field(description="An OAuth2.0 Web Client Secret.")),
+      ]
+    )
+
+  def fields(self) -> Sequence[str]:
+    return CM_CONVERSION_FIELDS
+
+  def batch_size(self) -> int:
+    return 10000
```
Comment (caiotomazelli) [inline dags/destinations/cm360oci.py:245]: I think this should be 1000 according to https://developers.google.com/doubleclick-advertisers/guides/conversions_faq?

File: sample_data/cm360_conversions_sample_data.csvh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+floodlightConfigurationId,floodlightActivityId,encryptedUserId,mobileDeviceId,timestamp_micros,value,quantity,ordinal,limitAdTracking,childDirectedTreatment,gclid,nonPersonalizedAd,treatmentForUnderage,matchId,dclid,impressionId,userIdentifiers,conversion_kind
+13641571,16352759,"test","test","",1695955940000000,"","","1","","","0000000000","","","","","","",""
```
Comment (caiotomazelli) [inline sample_data/cm360_conversions_sample_data.csvh:2]: Are all ids here mocked? I think it's best to remove any "real" data, even if not PII.

Comment (dandemel) [reply sample_data/cm360_conversions_sample_data.csvh:2]: Yes they are... they are from my own test campaign in dollar-a-day

File: dags/destinations/cm360oci.py (hunk 1 above)
Comment (dandemel) [reply dags/destinations/cm360oci.py]: fixed

**Review by caiotomazelli — APPROVED** [review]

**Pull request merged**